translations/translations.json

```diff
@@ -4794,8 +4794,8 @@
       "refresh": "Liste der nahegelegenen Regionen aktualisieren",
       "pageTitle": "Lokale Informationen für Dich",
       "metaDescription": "{{appName}} ist Dein digitaler Guide für Deutschland. Finde lokale Informationen, Veranstaltungen und Beratung. Immer aktuell und in Deiner Sprache.",
-      "cityNotFound": "Deine Region nicht gefunden?",
-      "clickHere": "Hier klicken"
+      "cityNotFound": "Deine Region nicht gefunden? Werde jetzt aktiv!",
```

Comment:
:wrench: I think we should be consistent with the exclamation mark between the different languages. Either we should use it everywhere or nowhere.

Reply:
I did that on purpose because in English, exclamation marks after an imperative are really only for a direct order that someone is yelling. But in German, it's common to have an exclamation mark after an imperative even when it's just a request. So how about I add a period to the English one?

Reply:
Sounds good